### PR TITLE
Creation of a email fill in reminder web service

### PR DESCRIPTION
**Pull request by pabelenda**
To implement what is stated in #334 

 * Added configuration parameters
 * Added a couple of email templates
 * Initial version of the web service

### CODE REVIEW

**Comment by jaragunde** [comment]
It looks good, but I've realized that any user would be able to trigger this service by opening the corresponding url `http://path/to/phpreport/web/services/sendEmailAlertNoFill.php`.

What means do we have to prevent web users to abuse this script? Maybe we could put it in a specific directory, and protect the web server from opening it? I suggest a `/jobs` directory in the repository root.

**Comment by pabelenda** [comment]
I've moved the file and added a new email type: the last reminder before closing the month.

File: jobs/sendEmailAlertNoFill.php [hunk 1]
```diff
@@ -74,80 +72,43 @@ function sendEmail($email, $lastTaskDate, $from, $cc, $subject, $template) {
...
-                    sendEmail($email, $lastTaskDate, $from, $cc, $subject, $template);
-                } elseif ($difference >= $NO_FILL_DAYS_TRIGGER_WARNING) {
-                    array_push($warnedUsers, $login);
-                    $cc = $NO_FILL_CC_WARNING;
-                    $subject = $NO_FILL_SUBJECT_WARNING;
-                    $template = $NO_FILL_TEMPLATE_WARNING;
-                    sendEmail($email, $lastTaskDate, $from, $cc, $subject, $template);
+        if (in_array($user->getLogin(), $excludedUsers))
+            continue;
+        else {
```
Comment (jaragunde) [inline jobs/sendEmailAlertNoFill.php]: A couple of nits here:
* If you use `continue` , you don't need the else branch. Or you could have written the condition as `!in_array` with no else branch.
* I'd have used curly braces to reduce the ambiguity.

File: jobs/sendEmailSummaryNoFill.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+    $users = UsersFacade::GetAllUsers();
+    $excludedUsers = [];
+    $criticalWarnedUsers = [];
+    $lastWarnedUsers = [];
+
+    foreach ($users as $user) {
+        if (in_array($user->getLogin(), $excludedUsers))
+            continue;
+        else {
```
Comment (jaragunde) [inline jobs/sendEmailSummaryNoFill.php]: Same here about if, else, continue.

**Comment by pabelenda** [comment]
I've gathered all the changes into one single commit.

I've also changed what you suggested. I think it is better looking now, right?

**Pull request merged**